payjoin-cli/src/app/v2/ohttp.rs

```diff
@@ -119,6 +110,11 @@ pub(crate) async fn validate_relay(
     let payjoin_directory = config.v2()?.pj_directory.clone();
     let relays = config.v2()?.ohttp_relays.clone();
 
+    //Local relays are incapable of acting as proxies so we must opportunistically fetch keys from the config
+    if let Some(relay) = config.v2()?.ohttp_relays.first().cloned() {
+        return Ok(relay);
+    }
+
```

Comment:
I think we discussed removing this and letting the loop run

Reply:
I'm assuing the directory doesn't accept a localhost relay... but that should be the localhost directory... let me look into this a little more

Reply:
Rather than work around this, does it make more sense to have the test use the directory from the bip21? Seems incorrect for a sender to use a default over what is provided in the bip21.